actors/system/src/lib.rs

```diff
@@ -25,9 +25,25 @@ pub enum Method {
 /// System actor state.
 #[derive(Default, Deserialize, Serialize)]
 #[serde(transparent)]
-pub struct State([(); 0]);
+pub struct State {
+    // builtin actor registry: Vec<(String, Cid)>
+    builtin_actors: Cid,
+}
 impl Cbor for State {}
 
+impl State {
+    pub fn get_builtin_actors<B: Blockstore>(
+        &self,
+        store: &B,
+    ) -> Result<Vec<(String, Cid)>, String> {
+        match store.get_cbor(&self.builtin_actors) {
+            Ok(Some(obj)) => Ok(obj),
+            Ok(None) => Err("failed to load builtin actor registry; not found".to_string()),
+            Err(e) => Err(e.to_string()),
+        }
+    }
```

Comment:
If you use `anyhow` (and import `anyhow::Context`), you can write this as:

```suggestion
        store.get_cbor(&self.builtin_actors)?
            .context("failed to load builtin actor registry; not found")
    }
```

Reply:
ok. What exactly is this context incantation?

Reply:
It attaches a message to an error, kind of like `fmt.Errorf("was doing X and this happened: %w")`.